frontend/src/pages/PipelineDetails.tsx

```diff
@@ -151,77 +157,77 @@ class PipelineDetails extends Page<{}, PipelineDetailsState> {
-                      <div className={css.summaryKey}>Description</div>
-                      <div>{pipeline.description}</div>
-                    </Paper>
-                  )}
+                    </div>
+                    <div className={css.summaryKey}>Uploaded on</div>
+                    <div>{formatDateString(pipeline.created_at)}</div>
+                    <div className={css.summaryKey}>Description</div>
+                    <div>{pipeline.description}</div>
+                  </Paper>
+                )}
 
-                  <Graph graph={this.state.graph} selectedNodeId={selectedNodeId}
-                    onClick={id => this.setStateSafe({ selectedNodeId: id })} />
+                <Graph graph={this.state.graph} selectedNodeId={selectedNodeId}
+                  onClick={id => this.setStateSafe({ selectedNodeId: id })} />
 
-                  <SidePanel isOpen={!!selectedNodeId}
-                    title={selectedNodeId} onClose={() => this.setStateSafe({ selectedNodeId: '' })}>
-                    <div className={commonCss.page}>
-                      {!selectedNodeInfo && <div>Unable to retrieve node info</div>}
-                      {!!selectedNodeInfo && <div className={padding(20, 'lr')}>
-                        <StaticNodeDetails nodeInfo={selectedNodeInfo} />
-                      </div>}
-                    </div>
-                  </SidePanel>
-                  <div className={css.footer}>
-                    {!summaryShown && (
-                      <Button onClick={() => this.setStateSafe({ summaryShown: !summaryShown })} color='secondary'>
-                        Show summary
-                      </Button>
+                <SidePanel isOpen={!!selectedNodeId}
+                  title={selectedNodeId} onClose={() => this.setStateSafe({ selectedNodeId: '' })}>
+                  <div className={commonCss.page}>
+                    {!selectedNodeInfo && (
+                      <div className={commonCss.absoluteCenter}>Unable to retrieve node info</div>
                     )}
-                    <div className={classes(commonCss.flex, summaryShown && css.footerInfoOffset)}>
-                      <InfoIcon style={{ color: color.lowContrast, height: 16, width: 16 }} />
-                      <span className={css.infoSpan}>Static pipeline graph</span>
-                    </div>
+                    {!!selectedNodeInfo && <div className={padding(20, 'lr')}>
+                      <StaticNodeDetails nodeInfo={selectedNodeInfo} />
+                    </div>}
+                  </div>
+                </SidePanel>
+                <div className={css.footer}>
+                  {!summaryShown && (
+                    <Button onClick={() => this.setStateSafe({ summaryShown: !summaryShown })} color='secondary'>
+                      Show summary
+                      </Button>
+                  )}
+                  <div className={classes(commonCss.flex, summaryShown && css.footerInfoOffset)}>
```

Comment:
`summaryShown && css.footerInfoOffset` should either have an additional boolean for when there is no summary (for embedded pipelines), or `summaryShown` should just be set to false in that case. Maybe in `load()` ?

Reply:
Yup, added a `!!pipeline` condition in there, that was the missing piece to shift it back.